cmd/zenchron-engineering/selfhost.go

```
@@ -620,6 +621,12 @@ func validCodexExecution(execution codexExecution) bool {
 	return execution.Provider != "" && execution.Model != "" && (execution.AuthMode == "chatgpt" || execution.AuthMode == "api") && execution.Attempt > 0 && execution.MaxAttempts >= execution.Attempt
 }
 
+// writeSelfhostPreflight records the trusted repository and base selected for
+// this run before an executor is invoked or a candidate is published.
+func writeSelfhostPreflight(stdout io.Writer, identity, base string, runtime goRuntime) {
+	fmt.Fprintf(stdout, "Selfhost preflight: repository %s; trusted origin/main base %s; Go runtime %s\n", identity, base, runtime)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)